src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -653,6 +653,29 @@ def is_mapper_device(device_name):
     return device_name.startswith(('/dev/mapper', '/dev/dm-'))
 
 
+def is_writable_raw_device(disk_path):
+    '''
+    Can we open the device in exclusive mode ?
+    If not, someone is using it like a raw database
+    '''
+    open_flags = (os.O_RDWR | os.O_EXCL)
+    open_mode = 0
+    open_disk = disk_path
```

Comment:
this assignment doesn't seem like it accomplishes anything, did you had an idea that changed here? Using `disk_path` is fine I think

Reply:
yup. Stupid copy/paste. Fixed.